jira_agile_metrics/querymanager.py

```diff
@@ -81,14 +96,17 @@ def __init__(self, jira, settings):
             self.fields_to_attributes[field_id] = name
 
     def field_name_to_id(self, name):
+        arr_name = name.split(".")
```

Comment:
Because "team" actually gets translated to a customfield by the jira API. The old behaviour might change `team` to `customfield_10800`

This new code needs to translate `team.name` to `customfield_10800.name`